x-pack/test/functional/apps/ml/short_tests/settings/calendar_creation.ts

```diff
@@ -180,7 +180,7 @@ export default function ({ getService }: FtrProviderContext) {
       await asyncForEach(
         [automatedConfig, multiMetricConfig],
         // @ts-expect-error not full interface
-        async (config) => await ml.api.createAnomalyDetectionJob(config)
+        async (config) => ml.api.createAnomalyDetectionJob(config)
```

Comment:
This case doesn't need `async` anymore
```suggestion
       (config) => ml.api.createAnomalyDetectionJob(config)
```

Reply:
@dmlemeshko Can you explain why the `await` was removed in this case? I'm seeing the same pattern quite a lot across tests so wondering why it was done just in this case.

Reply:
I believe we don't need `await` here since `createAnomalyDetectionJob` returns Promise, that will be managed by caller (`asyncForEach`)

```
export async function asyncForEach<T>(
  array: T[],
  callback: (item: T, index: number) => Promise<void>
) {
  for (let index = 0; index < array.length; index++) {
    await callback(array[index], index);
  }
}
```

Reply:
nice, that explains it, thanks!